app/src/processing/app/contrib/ContributionListing.java

```diff
@@ -526,18 +526,7 @@ String getLatestVersion(Contribution contribution) {
     if(newestContrib == null){
       return null;
     }
-    String latestVersion = newestContrib.getPrettyVersion();
-    if (latestVersion != null && !latestVersion.isEmpty()) {
-      if (latestVersion.toLowerCase().startsWith("build")) // For Python mode
-        return ("v" + latestVersion.substring(5, latestVersion.indexOf(','))
-            .trim());
-      else if (latestVersion.toLowerCase().startsWith("v")) // For ketai library
-        return latestVersion;
-      else
-        return ("v" + latestVersion);
-    }
-    else
-      return null;
+    return newestContrib.getPrettyVersionShort();
```

Comment:
No need for a whole function here... Just use the `prettyVersion` and set the max size of the field. When the authors see their stuff looks goofy, that's a motivation to fix it.